integration/integrate-with-splunk.adoc

```diff
@@ -0,0 +1,52 @@
+[id="integrate-with-splunk"]
+= Integrating with Splunk
+include::modules/common-attributes.adoc[]
+:context: integrate-with-splunk
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://www.splunk.com/[Splunk], you can forward alerts from {product-title} to Splunk and view vulnerability and compliance related data from within Splunk.
+
+Depending on your use case, you can integrate {product-title} with Splunk by using the following ways:
+
+* By xref:../integration/integrate-with-splunk.adoc#integrate-splunk-http-event-collector[using an HTTP event collector] in Splunk
+** Use the event collector option to forward alerts and audit log data.
+* By xref:../integration/integrate-with-splunk.adoc#integrate-splunk-add-on[using the StackRox Kubernetes Security Platform add-on].
+** Use the add-on to pull vulnerability detection and compliance data into Splunk
```

Comment:
The punctuation still isn't consistent here. You removed it from this subbullet, but the above subbullet has a period. And one of the main bullets uses a period but the other doesn't.